v1.18/qfusion/README.md

```diff
@@ -0,0 +1,17 @@
+## QFusion
+QFusion is an enterprise-level DBaaS platform based on cloud native technologies which support for MySQL,SQL Server,Redis,Oracle,PostgreSQL and MongoDB database.It implements key components of database operator orchestration and provides self-healing and auto-scale database feature
+
+## Reproduce Conformance Tests
+###  prepare
+`contact [Woqutech](http://woqutech.com)  to download relvent images and push to own registry`
```

Comment:
Please include installation instructions, please include detailed instructions for installing/configuring your product - they cannot just include a link to their instructions (to minimize link rot etc)

Reply:
Following up on request for installation instructions.

Reply:
installation instruction update in README.md